src/resources/ECFormLinks.ts

```diff
@@ -31,6 +31,8 @@ export type ECFormLinkItem = {
     tokenType: TransactionTokenType;
     tokenOnly: boolean;
     hideCvv?: boolean;
+    email?: string;
```

Comment:
We do not have that. We will have to rely on the link metadata for this parameter. We only need to add `hidePrefilledEmail` 🙂

Reply:
Removed :+1: Please check again :bow: